libs/utils/decorate.js

```diff
@@ -68,5 +53,61 @@ export function decorateBlockBg(block, node) {
 export function getBlockSize(el, defaultSize = 1) {
   const sizes = ['small', 'medium', 'large', 'xlarge'];
   if (defaultSize < 0 || defaultSize > sizes.length - 1) return null;
-  return sizes.find((size) => el.classList.contains(size)) || sizes[defaultSize];
+  return sizes.find((size) => el?.classList.contains(size)) || sizes[defaultSize];
+}
+
+function getDescendants(node, fragment = document.createDocumentFragment()) {
+  for (let i = 0; i < node.childNodes.length; i += 1) {
+    const child = node.childNodes[i];
+    fragment.appendChild(child);
+    getDescendants(child, fragment.lastChild);
+  }
+  return fragment;
+}
+
+export function decorateButtons(buttons) {
+  const mapBtnSize = { large: 'button-L', xlarge: 'button-XL' };
+  buttons.forEach((button) => {
+    const block = button.closest('.section div[class]:not(.content)');
+    const blockSize = getBlockSize(block);
+    const size = mapBtnSize[blockSize] ?? blockSize;
+    const parent = button.parentElement;
+
+    const child = button.childNodes?.length > 0 ? Array.from(button.childNodes).filter((n) => n.nodeName === 'STRONG' || n.nodeName === 'EM')[0] : null;
```

Comment:
nit: line too long, break up.  Possibly also use `isStrongOrEm` here (ok to redefine as not worth to import from utils)

Reply:
done!